streams/src/test/java/org/apache/kafka/streams/integration/StoreQueryIntegrationTest.java

```diff
@@ -650,12 +659,12 @@ private void produceValueRange(final int key, final int start, final int endExcl
     }
 
     private Properties streamsConfiguration() {
-        final String safeTestName = safeUniqueTestName(getClass(), testName);
+        final String safeTestName = safeUniqueTestName(getClass(), this.testInfo);
```

Comment:
Instead of storing `testInfo` as field, you could store `applicationld` as field and compute the `applicationId` in `before()`. In such a way you only need to compute the `applicationId` once instead of twice for each test with two Streams clients.
See https://github.com/apache/kafka/blob/c8675d4723a45d99aee6f64c6bf4703eeaea81aa/streams/src/test/java/org/apache/kafka/streams/integration/RestoreIntegrationTest.java#L124 for an example.

Reply:
done.